spec/factories/parents.rb

```diff
@@ -11,32 +11,19 @@
 #  name                         :string
 #  phone                        :string
 #  recorded_at                  :datetime
-#  relationship                 :integer
-#  relationship_other           :string
 #  created_at                   :datetime         not null
 #  updated_at                   :datetime         not null
 #
 FactoryBot.define do
   factory :parent do
     transient do
-      random { Random.new }
```

Comment:
Think we had a reason to allow passing in a transient `random:` attribute...

Reply:
I did wonder about that, but then I couldn't see it was being used anywhere so I figured we could put it back if we needed to.